recipes-devtools/rust/libstd-rs.inc

```diff
@@ -20,5 +20,6 @@ do_compile_prepend () {
 do_install () {
     mkdir -p ${D}${rustlibdir}
 
+    rm ${B}/${TARGET_SYS}/release/deps/*.d
```

Comment:
We could use a comment here to explain why we do this.

Reply:
I can add a comment in the file for this. Until I get that added,  with the incremental build support added in 1.24, the libstd deps directory also includes dependency files that get installed. Those are really only needed to incrementally rebuild the libstd library itself and don't need to be installed.

Reply:
Got it now.